django/contrib/admin/options.py

```diff
@@ -566,9 +567,9 @@ def media(self):
         extra = '' if settings.DEBUG else '.min'
         js = [
             'core.js',
-            'admin/RelatedObjectLookups.js',
             'vendor/jquery/jquery%s.js' % extra,
             'jquery.init.js',
+            'admin/RelatedObjectLookups.js',
```

Comment:
This file now depends on jQuery https://github.com/django/django/pull/5567/files#diff-ec17958cc9f7c2542303557dfc4ad997R169